mail_tracking/demo/demo.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="True">
```

Comment:
It's the contrary. Demo data should be noupdate. Examples:

https://github.com/odoo/odoo/blob/16.0/addons/sale/data/product_demo.xml#L2
https://github.com/odoo/odoo/blob/16.0/addons/purchase/data/purchase_demo.xml#L3

Reply:
Demo data is noupdate by default, that's why it shouldn't be marked. Pleasew read guidelines.

Reply:
<odoo>: for noupdate=0 or demo data (demo data is non-updatable by default)

([Source](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id20))

But I agree its not a blocking point, so either way is OK to me.

Reply:
Thanks for the reference, although is not directly linkable (I have searched the term):

![Selección_018](https://github.com/OCA/social/assets/7165771/005e9789-7782-4a9c-b7cc-2e2b0428ac31)

I have never tried such behavior, and all demo data I have seen includes the noupdate part. If it's OK for you, let's keep for clarity (the same I keep `position="inside"` on inherited views for clarity).

Reply:
> I have never tried such behavior, and all demo data I have seen includes the noupdate

I have a different appreciation regarding that. Almost all demo data I've seen don't include the noupdate attribute.

The reason I consider it shouldn't be included (I wrote that part myself) is that, in my understanding,  noupdate makes no sense under a demo data context. Demo data is never updated because when updating the module, demo files are not even processed, which means a `-i module` is required; which would update also noupdate records anyway.